python/practices/practice-3/src/path1/long_words_printer.py

```diff
@@ -0,0 +1,32 @@
+"""
+Check and print words with more than 20 letters from a word file
+"""
+
+
+import os
+
+
+def print_words(file):
```

Comment:
Please refactor this function, It only needs to accept the content of a file instead of the file name.

Reply:
Done